xmppserver/src/main/java/org/jivesoftware/openfire/net/ServerStanzaHandler.java

```diff
@@ -132,6 +150,48 @@ void startTLS() throws Exception {
         //needed ? Connection.ClientAuth.needed : Connection.ClientAuth.wanted
         connection.startTLS(false, false);
     }
+
+    @Override
+    protected String getStreamHeader() {
+        StringBuilder sb = new StringBuilder(200);
```

Comment:
Nit: There's no need to initialize the capacity of the StringBuilder. The theoretical improvement in speed that this gives is negated by the fact that changes in the variables (and/or future code modifications that neglect to update this value) won't make this value ever 'correct'. The biggest impact of this is that it might confuse someone in the future.

Maybe even replace this entire ting with simple String-concatination? An IDE should help you with that.

Reply:
fixed